cmake/DownloadProject.CMakeLists.autogen.in

```diff
@@ -0,0 +1,19 @@
+# Distributed under the OSI-approved MIT License.  See accompanying
+# file LICENSE or https://github.com/Crascit/DownloadProject for details.
```

Comment:
Weird that Crascit doesn't have the normal MIT license header. I guess we leave it rather than put the real MIT license text.